src/lib/generate.js

```diff
@@ -0,0 +1,34 @@
+import TemplateStyleSheet from './template-stylesheet.json'
+
+let GENERATED_FILE = null
```

Comment:
Dion't use `CONSTANT_CASE` if it's not a true constant^ 

`generatedFile` would suffice